backend/middlewares/auth/index.js

```diff
@@ -0,0 +1,17 @@
+const initLocals = (req, res, next) => {
+	res.locals = {
+		...res.locals,
+		inData: {},
+		outData: {},
+	};
+	next();
+};
+
+const authorize = async (req, res, next) => {
+	next();
```

Comment:
Nice, we'll fix auth later. Me and @lucas-larsson discussed adding a table for cookies in the db which we can write to the Cookie header field later.